spec/book_spec.rb

```diff
@@ -0,0 +1,11 @@
+require './book'
+
+describe Book do
+  context 'Creating a book' do
+    it 'Is a book created?' do
+      book = Book.new('Saung', 'Hnin')
+      expect(book.title).to eq('Saung')
+      expect(book.author).to eq('Hnin')
+    end
+  end
+end
```

Comment:
- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md#:~:text=Add%20unit%20tests%20for,Book) there should be an 'it' block for each class method (at least). Please make sure to create a test block fo r`add_rental` method in the book class.